_gsocblogs/2022/blog_ATHENA_UjwalKundur.md

```diff
@@ -0,0 +1,73 @@
+---
+project: ATHENA
+title: A foray into HPC and Hardware Accelerators
+author: Ujwal Kundur
+date: 22.07.2022
+year: 2022
+layout: blog_post
+logo: ATHENA-logo.png
+intro: |
+  Parallelizing the IslandCluster Algorithm for the EIC ECAL, a progress report.
+---
+
+## Introduction
+
+Hello readers, I'm Ujwal and this is my first blogpost ever. Is this how YouTubers feel like when they talk in front of a camera? :)
+
+This post is meant to be a progress update to my mentors, amusing read for the physicist-developer and absolute nonsense for the uninitiated. Nevertheless, I hope this is accessible to the interested layman without sacrificing the technical accuracy so coveted in Physics.
+
+The project is pretty straight-forward, at least on the surface; Porting a clustering algorithm used in the reconstruction of particle collision events at the Electron-Ion Collider (**EIC**) to **SYCL**.
+The clustering algorithm needs to scale in order to process increasing number of hits as the Sensor resolution is upgraded over the years, this requires hardware acceleration which is already being used significantly in AI/ML to train models using GPUs. Other hardware accelerators do exist, for example, FPGAs and ASICs (Application Specific Integrated Circuits) which are prominently used in Bitcoin-mining. We stick to GPUs in this project.
+
+SYCL aims to make programmers' lives easier by allowing us to target all these different hardware architectures from a single codebase. While many implementations of the SYCL standard exist, similar to how there are many implementations of *ISO C*, Intel's offering - **Intel OneAPI DPC++** (Data Parallel C++) seems to be the most mature project and we stick to it for our SYCL needs.
```

Comment:
lifes

Reply:
Hi @hegner I don't think that's a typo ( I did a proof-read before the PR ). https://www.queens-english-society.com/lives-vs-lifes
![image](https://user-images.githubusercontent.com/54140487/180725990-f79b8160-65dd-4606-a2e9-3322911f1911.png)

Reply:
However, Lemme know if I have to correct it

Reply:
You are correct!